dotCMS/src/integration-test/java/com/liferay/portal/ejb/UserPersistenceTest.java

```diff
@@ -0,0 +1,219 @@
+package com.liferay.portal.ejb;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.liferay.portal.model.User;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Unit test for the {@link UserPersistence}
+ */
+public class UserPersistenceTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)